deployments/cloudfoundry/tile/tile.yml

```diff
@@ -94,16 +94,16 @@ forms:
   - name: otel_proxy_http
     type: string
     label: HTTP proxy URL
-    description: URL used as the proxy URL for HTTP requests unless overridden by otel_proxy_no
+    description: URL used as the proxy URL for HTTP requests unless overridden by "Proxy exclusions"
```

Comment:
This is in the Tanzu Tile UI, so the user wouldn't know what `otel_proxy_no` is referring to. The text box is labelled `Proxy exclusions`, so this makes the text box's description consistent with the UI's label.